backend/database.sql

```diff
@@ -0,0 +1,78 @@
+-- phpMyAdmin SQL Dump
```

Comment:
I don't see the branch and department schema in this database file. Please update.

Reply:
sorry..done